src/NServiceBus.Transport.RabbitMQ.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -107,6 +107,75 @@ namespace NServiceBus
             "edException. Will be removed in version 8.0.0.", true)]
         public static NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> UseExternalAuthMechanism(this NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> transportExtensions) { }
     }
+    public static class RabbitMqTransportApiExtensions
+    {
+        [System.Obsolete("Use `EndpointConfiguration.UseTransport(TransportDefinition)` instead. Will be tr" +
+            "eated as an error from version 8.0.0. Will be removed in version 9.0.0.", false)]
+        public static NServiceBus.RabbitMqTransportLegacySettings UseTransport<T>(this NServiceBus.EndpointConfiguration config)
+            where T : NServiceBus.RabbitMQTransport { }
+    }
+    public class RabbitMqTransportLegacySettings : NServiceBus.TransportSettings<NServiceBus.RabbitMQTransport>
```

Comment:
are the `CustomMessageIdStrategy` and the `DelayedDelivery` setting missing here?

Reply:
CustomMessageIdStrategy is there in line 126.

DelayedDelivery was missing, I'll add it.
```
public static DelayedDeliverySettings DelayedDelivery(this TransportExtensions<RabbitMQTransport> transportExtensions)
```